resources/tentative/charts/chartjs.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Chart.js</title>
+  </head>
+  <body>
+    <div id="app">
+      <button id='run-all' type='button'>Run all scenario (development only)</button>
+      <button id="prepare" type="button">Prepare data</button>
+      <button id="add-scatter-chart-button" type="button">Add a scatter chart</button>
+      <button id="open-tooltip" type="button">Open tooltips</button>
+      <button id="reset" type="button">Reset</button>
+    </div>
+    <canvas id="chart"></canvas>
+    <script type="module" src="/chartjs.js"></script>
```

Comment:
Can we use a relative path instead?

Reply:
The Vite bundler makes it relative in the dist files, so I think we're good. Please tell me if I'm missing something :-)

Reply:
Ah, this is because we specified `base: "./"` in the configuration for vite. I just copy pasted the configuration from the `editors` workload without thinking about it too much, but this looks like the good thing to do!

Reply:
Looks like it's still not using a relative path?

Reply:
It is in the `dist` files :-) https://github.com/WebKit/Speedometer/pull/149/files#diff-39eaea024fee1f3009903ff9b4f74e887b7c5091a3d0d33d00398da2d440c1ce
As I explained above, `vite` configuration does it at build time.